app/src/main/res/layout/browser_overlay.xml

```diff
@@ -72,6 +84,25 @@
             android:lines="1"
             android:selectAllOnFocus="true"
             android:importantForAutofill="no"/>
+
+        <View
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:layout_weight="1" />
+
+        <ImageView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_weight="0"
+            android:tint="@android:color/white"
```

Comment:
This is white while the text looks to be some sort of shade of gray and it looks inconsistent: does the design dictate these should be the same color?